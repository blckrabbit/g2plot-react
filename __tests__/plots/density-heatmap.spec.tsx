import React from 'react'
import { create } from 'react-test-renderer'
import DensityHeatmapChart from '../../src/plots/density-heatmap'

describe('DensityHeatmapChart', () => {
  test('should render without crashed', () => {
    const renderer = create(<DensityHeatmapChart data={[]} colorField="x" />)

    expect(renderer.toJSON()).toMatchSnapshot()
  })
})
