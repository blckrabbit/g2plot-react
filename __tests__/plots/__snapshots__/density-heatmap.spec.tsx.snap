// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DensityHeatmapChart should render without crashed 1`] = `<div />`;
